addons/sms/data/mail_demo.xml

```diff
@@ -49,10 +49,19 @@
         <field name="res_partner_id" ref="base.res_partner_address_28"/>
         <field name="notification_type">sms</field>
         <field name="notification_status">exception</field>
-        <field name="failure_type">sms_credit</field>
+        <field name="failure_type">sms_acc</field>
     </record>
-    <record id="message_demo_partner_1_3_notif_1" model="mail.notification">
```

Comment:
Just to be annoying, we could add a demo data with a sent sms :D .

Reply:
All done :)